src/domain/collaboration/canvas/containers/CanvasProvider.tsx

```diff
@@ -22,7 +22,7 @@ export type TemplateQuery = {
 } & { hubId: string };
 
 export interface IProvidedEntities {
-  canvases: CanvasFragmentWithCallout[];
+  canvas: CanvasDetailsFragment | undefined;
   templates: CreateCanvasCanvasTemplateFragment[];
   calloutId: string | undefined;
   authorization: NonNullable<CollaborationWithCanvasDetailsFragment['callouts']>[0]['authorization'];
```

Comment:
I think, if you get to get rid of [hub/ch/opp]Canvases queries (see the other comment) you'll be able to remove CollaborationWithCanvasDetails fragment and this one can become `CanvasDetailsFragment['authorization']` 
** I think **

Reply:
I cannot remove HubCanvases, it is still used and it uses CollaborationWithCanvasDetails.